tools/emscripten.py

```diff
@@ -407,10 +408,6 @@ def emscript(in_wasm, out_wasm, outfile_js, js_syms, finalize=True):
 
   report_missing_symbols(forwarded_json['librarySymbols'])
```

Comment:
If all the missing symbol reporting is done by wasm-ld, do we need this line?

Reply:
Can we not remove this line in this PR, though, if that's the case? Or is that why this is still marked draft?

Reply:
I think because report_missing_symbols is currently where we report symbols that are part of `EXPORTED_FUNCTIONS` but were not actually exported.

Currently we use `--export-if-defined` for these symbols which means the linker doesn't fail.. but I'm planning on changing that.